docs/releases.md

```diff
@@ -1,5 +1,26 @@
 # Releases
 
+## 1.7
+[Full Release Notes](https://github.com/securefederatedai/openfl/releases/tag/v1.7)
+
+### New Features
+- [**FederatedRuntime**](https://openfl.readthedocs.io/en/latest/about/features_index/workflowinterface.html#runtimes-future-plans) for Workflow API: enables a seamless transition from a local simulation (via LocalRuntime) to a distributed Federated Learning deployment. Check out the [FederatedRuntime 101 Tutorial](https://github.com/securefederatedai/openfl/tree/develop/openfl-tutorials/experimental/workflow/FederatedRuntime/101_MNIST) to try it yourself.
+
+- [**Federated XGBoost**](https://github.com/securefederatedai/openfl/tree/develop/openfl-workspace/xgb_higgs): Adding support for XGBoost in OpenFL via TaskRunner API, illustrated with the Higgs dataset.
+
+- [**Callbacks**](https://openfl.readthedocs.io/en/latest/openfl.callbacks.html): An abstraction for running user-defined actions in TaskRunner API or Workflow API. Callbacks can be used to perform custom actions at different stages of the Federated Learning process.
+
+### Enhanced Developer Experience

```

Comment:
+New documentation.

Reply:
Do you mean the revamping of the documentation site?